avocado/utils/disk.py

```diff
@@ -311,3 +312,101 @@ def get_disk_mountpoint(device):
             if dev == device:
                 return fs_dir
         return None
+
+
+def create_linux_raw_partition(disk_name, size=None, num_of_par=1):
+    """
+    Creates partitions using sfdisk command
+
+    :param disk_name: disk/device name
+    :type disk_name: str
+    :param size: size of partition
+    :type size: str
+    :param num_of_par: Number of partitions to be created
+    :type num_of_par: int
+
+    Returns list of created partitions
+    """
+    if not size:
+        size = get_size_of_disk(disk_name) / 1024 / 1024 / 1024
+        size = size / num_of_par
+        size = str(size) + "G"
```

Comment:
so if my disk size is 1TB,  size  = 1TB/(1024 * 3) --> instead of /1024/1024/1024  ? just think if feasible ?

Reply:
@maramsmurthy a check for smaller disks also needed.. I mean what if usb is 20G

Reply:
"get_size_of_disk" definition returns size in number of bytes. We are just converting it to GB's. Not performing any operation here